ext/pgsql/pgsql.stub.php

```diff
@@ -974,6 +974,9 @@ function pg_socket_poll($socket, int $read, int $write, int $timeout = -1): int
 #ifdef HAVE_PG_SET_CHUNKED_ROWS_SIZE
     function pg_set_chunked_rows_size(Pgsql\Connection $connection, int $size): bool {}
 #endif
+#ifdef HAVE_PG_CLOSE_STMT
+    function pg_close_stmt(Pgsql\Connection $connection, string $stmt_name): Pgsql\Result|false {}
```

Comment:
`$statement_name` is the consistent param name with the existing params.

```suggestion
    function pg_close_stmt(Pgsql\Connection $connection, string $statement_name): Pgsql\Result|false {}
```